ci/kind/test-e2e-kind.sh

```diff
@@ -22,8 +22,9 @@ function echoerr {
     >&2 echo "$@"
 }
 
-_usage="Usage: $0 [--encap-mode <mode>] [--no-proxy] [--np] [--coverage] [--help|-h]
+_usage="Usage: $0 [--encap-mode <mode>] [--ipfamily <ipfamily>] [--no-proxy] [--np] [--coverage] [--help|-h]
         --encap-mode                  Traffic encapsulation mode. (default is 'encap').
+        --ipfamily                    Configures the ipFamily for the KinD cluster.
```

Comment:
it's a nit but I would prefer `--ip-family <v4|v6>` since we already use it somewhere else

https://github.com/antrea-io/antrea/blob/9adf9c9120578b50730c7a160fe7d38e09dcda2c/test/e2e/infra/vagrant/provision.sh#L6